ion/skipper.go

```diff
@@ -657,7 +657,7 @@ func (t *tokenizer) skipContainer(term int) (int, error) {
 // char.
 func (t *tokenizer) skipContainerHelper(term int) error {
 	if term != ']' && term != ')' && term != '}' {
-		panic(fmt.Sprintf("unexpected character: %v. Expected one of the closing container characters: ] } )", string(term)))
+		panic(fmt.Sprintf("unexpected character: %v. Expected one of the closing container characters: ] } )", rune(term)))
```

Comment:
Is this what is wanted here? Or something more like the `%q` formatting directive with a bare `term`?

See: https://play.golang.org/p/JKRiykdCWJv

Reply:
```plain
Using string(term):
unexpected character: A. Expected one of the closing container characters: ] } )
Using rune(term):
unexpected character: 65. Expected one of the closing container characters: ] } )
Using string(rune(term)):
unexpected character: A. Expected one of the closing container characters: ] } )
Using %c with bare term:
unexpected character: A. Expected one of the closing container characters: ] } )
Using %q with bare term:
unexpected character: 'A'. Expected one of the closing container characters: ] } )
```

Reply:
Ah, good call! :+1: for `%q`.